hermes-engine.podspec

```diff
@@ -32,34 +29,4 @@ Pod::Spec.new do |spec|
   spec.osx.vendored_frameworks = "destroot/Library/Frameworks/macosx/hermes.framework"
 
   spec.xcconfig            = { "CLANG_CXX_LANGUAGE_STANDARD" => "c++14", "CLANG_CXX_LIBRARY" => "compiler-default", "GCC_PREPROCESSOR_DEFINITIONS" => "HERMES_ENABLE_DEBUGGER=1" }
-
-  unless ENV['hermes-artifact-url']
-    spec.prepare_command = <<-EOS
-      # When true, debug build will be used.
-      # See `build-apple-framework.sh` for details
-      DEBUG=#{HermesHelper::BUILD_TYPE == :debug}
-
-      # In a release package, there are no utilities and source files, we exit
-      # early as there is nothing to build 
-      if [ ! -f ./utils/build-apple-framework.sh ]; then
-        exit 0;
-      fi
-
-      # Source utilities into the scope
-      . ./utils/build-apple-framework.sh
-
-      # If universal framework for iOS does not exist, build one
-      if [ ! -d destroot/Library/Frameworks/iphoneos/hermes.framework ]; then
-        build_apple_framework "iphoneos" "armv7;armv7s;arm64" "#{HermesHelper::IOS_DEPLOYMENT_TARGET}"
-        build_apple_framework "iphonesimulator" "x86_64;i386" "#{HermesHelper::IOS_DEPLOYMENT_TARGET}"
-
-        create_universal_framework "iphoneos" "iphonesimulator"
-      fi
-
-      # If MacOS framework does not exist, build one
-      if [ ! -d destroot/Library/Frameworks/macosx/hermes.framework ]; then
-        build_apple_framework "macosx" "x86_64;arm64" "#{HermesHelper::OSX_DEPLOYMENT_TARGET}"
-      fi
-    EOS
-  end
```

Comment:
I think we should keep a `prepare_command` step so people can still build the pod directly from the repo, if needed.

Reply:
I was hoping that running `./utils/build-mac-framework.sh` will be enough to build directly from the repo, rather than keeping this + custom ENV to disable it in release versions. 

I am happy to keep it tho!

Reply:
When someone runs `pod install` they do not expect to have to manually invoke anything else, so let just do that for them by invoking the right script here?

Reply:
Sounds good to me.